cdci_data_analysis/analysis/parameters.py

```diff
@@ -586,6 +586,12 @@ class LongString(String):
 class Name(String):
     owl_uris = String.owl_uris + ("http://odahub.io/ontology#AstrophysicalObject",)
 
+class FileReference(String):
+    owl_uris = "http://odahub.io/ontology#FileReference"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
 class NumericParameter(Parameter):
```

Comment:
didn't we discuss that this can cause issues when the value is stored and passed further?

Reply:
which kind of issue as I dont remember...

Reply:
That this dict will be included in analysis_paramters, API code, product URLs.
Better to immediately introduce a parameter with an (constructed) URL which is also always valid way to pass this parameter.

Reply:
> That this dict will be included in analysis_paramters, API code, product URLs.
you mean the `kwargs` dict?

> Better to immediately introduce a parameter with an (constructed) URL which is also always valid way to pass this parameter.

this is what the `FileRefence` Parameter is for, perhaps it can be named as `ParameterURL`?

Reply:
Sorry I don't understand, what FileRefence is for?

http://odahub.io/ontology#FileReference already has a subclass with URL.

Reply:
If and when you represent the parameter as file URL, then FileURL seems appropriate.

Reply:
I introduced the FileURL parameter class which extends the FileReference, with the appropriate owl_uris